server/controllers/events/events_controller.go

```diff
@@ -43,6 +43,9 @@ const bitbucketCloudRequestIDHeader = "X-Request-UUID"
 const bitbucketServerRequestIDHeader = "X-Request-ID"
 const bitbucketServerSignatureHeader = "X-Hub-Signature"
 
+// The URL used for Azure DevOps test webhooks
```

Comment:
Has this PR been tested in your atlantis deployment? Were you able to verify that this change worked as expected?

Reply:
I'll build it and run through some tests - hold off until I've confirmed that's sorted

Reply:
Thank you! Once you confirm then we can merge

Reply:
@SSKLCP friendly ping

Reply:
@SSKLCP friendly ping

Reply:
@SSKLCP friendly ping